trick_source/sim_services/Executive/Executive_shutdown.cpp

```diff
@@ -101,9 +110,11 @@ int Trick::Executive::shutdown() {
             "     SIMULATION ELAPSED TIME: %12.3f\n"
             "        ACTUAL CPU TIME USED: %12.3f\n"
             "       SIMULATION / CPU TIME: %12.3f\n"
-            "     INITIALIZATION CPU TIME: %12.3f\n" ,
+            "     INITIALIZATION CPU TIME: %12.3f\n" 
+            "        SIMULATION RAM USAGE: %12.3fM\n"
```

Comment:
Good catch.Did put it there purposely for megabytes in case people are wondering. But we can remove it if not necessary or maybe should put MB to be more precise ;-)

Reply:
Either way works! I agree that MB might be more clear, though.

Reply:
Changed to MB :-)